administrator/components/com_modules/tmpl/select/default.php

```diff
@@ -18,16 +18,42 @@
 
 $function  = $app->input->getCmd('function');
 
+HTMLHelper::_('script', 'com_modules/admin-module-search.js', ['version' => 'auto', 'relative' => true, 'async' => 'async']);
+
 if ($function) :
 	HTMLHelper::_('script', 'com_modules/admin-select-modal.js', ['version' => 'auto', 'relative' => true]);
 endif;
 ?>
 <h2 class="mb-3"><?php echo Text::_('COM_MODULES_TYPE_CHOOSE'); ?></h2>
+
+<div class="container d-none" id="comModulesSelectSearchContainer">
+	<div class="row">
+		<div class="col-sm-4 offset-sm-4">
+			<div class="form-inline">
+				<label class="sr-only" for="comModulesSelectSearch">
+					<?= Text::_('COM_MODULES_TYPE_CHOOSE'); ?>
+				</label>
+				<div class="input-group mb-5 mr-sm-2">
+					<input type="text" value=""
+						   class="form-control-lg" id="comModulesSelectSearch"
```

Comment:
```suggestion
						   class="form-control" id="comModulesSelectSearch"
```

With that change the input and the icon are flush

![image](https://user-images.githubusercontent.com/1296369/83953958-c41d9a80-a83c-11ea-8082-e0dab1a7ba0a.png)

Reply:
Right. The reason I added the `-lg` was trying to make it prominent but that was before I realised I can make it autofocus AND accessible at the same time.